libs/features/pages/home/src/lib/features-pages-home.module.ts

```diff
@@ -8,9 +8,16 @@ import { HomePageComponent } from './main/home/home-page.component';
 import { NewsSectionComponent } from './components/news-section/news-section.component';
 
 import { HomeRoutingModule } from './home.routing';
+import { ConsultancyPageModule } from '@elewa-website/features/pages/consultancy-page';
 
```

Comment:
Same here, the order of imports; features imports should appear before local imports

Reply:
corrected